options/glibc/include/features.h

```diff
@@ -0,0 +1,4 @@
+#ifndef FEATURES_H
```

Comment:
What's the point of this header?

Reply:
This header was just a stub libcxx did not attempt to use anything from it, just expected it to be there

Reply:
Hmm... Maybe leave a comment stating that then.